<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    font: bold;
   border: 2px solid rgb(100, 100, 100);
    border-radius: 9px;
   margin-top: 6px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
     margin-left: auto;
     margin-right: auto;

   
}</string>
      </property>
      <property name="title">
       <string>Connexion</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_host">
             <property name="text">
              <string>Host</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_host">
             <property name="text">
              <string>10.0.4.203</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_port">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_topic">
             <property name="text">
              <string>Topic</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_topic">
             <property name="text">
              <string>capteur</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_port">
             <property name="maximum">
              <number>20000</number>
             </property>
             <property name="value">
              <number>1883</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_conect">
             <property name="styleSheet">
              <string notr="true">QPushButton {
color: white;
     background-color: green;
 border: 2px solid black;
     border-radius: 9px;
}</string>
             </property>
             <property name="text">
              <string>Connexion</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_souscrire">
             <property name="styleSheet">
              <string notr="true">QPushButton {
color: white;
     background-color: #2A9183;
 border: 2px solid black;
     border-radius: 9px;
}</string>
             </property>
             <property name="text">
              <string>Souscrire</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_quitter">
             <property name="styleSheet">
              <string notr="true">QPushButton {
color: white;
     background-color: red;
 border: 2px solid black;
     border-radius: 9px;
}</string>
             </property>
             <property name="text">
              <string>Quitter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
